### PR TITLE
ci: Associate recipe owners

### PR DESCRIPTION
**Pull request by thomasdhc**
# What does this PR do ?

Create config for recipe owner in each recipe for llm_finetune, vlm_finetune

# Changelog

- Add specific line by line info of high level changes in this PR.

# Before your PR is "Ready for review"

**Pre checks**:

- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA-NeMo/Automodel/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?

If you haven't finished some of the above items you can still open "Draft" PR.

# Additional Information

- Related to # (issue)

### CODE REVIEW

File: examples/llm_finetune/qwen/qwen1_5_moe_a2_7b_qlora.yaml [hunk 1]
```diff
@@ -99,3 +99,6 @@ optimizer:
   lr: 1.0e-4
   weight_decay: 0.0
   betas: [0.9, 0.95]
+
+ci:
+  recipe_owner: yuhezhang-ai
```
Comment (akoumpa) [inline examples/llm_finetune/qwen/qwen1_5_moe_a2_7b_qlora.yaml]:
```suggestion
  recipe_owner: adil-a
```

File: examples/llm_finetune/qwen/qwen2_5_0p5b_instruct_fineproofs_chat.yaml [hunk 2]
```diff
@@ -120,3 +120,6 @@ lr_scheduler:
 #   entity: <your_wandb_entity>
 #   name: <your_wandb_exp_name>
 #   save_dir: <your_wandb_save_dir>
+
+ci:
+  recipe_owner: zeel2104
```
Comment (akoumpa) [inline examples/llm_finetune/qwen/qwen2_5_0p5b_instruct_fineproofs_chat.yaml]:
```suggestion
  recipe_owner: adil-a
```

File: examples/llm_finetune/qwen/qwen2_5_7b_hellaswag_fp8.yaml [hunk 3]
```diff
@@ -114,4 +114,7 @@ optimizer:
 #   project: <your_wandb_project>
 #   entity: <your_wandb_entity>
 #   name: <your_wandb_exp_name>
-#   save_dir: <your_wandb_save_dir> 
\ No newline at end of file
+#   save_dir: <your_wandb_save_dir>
+
+ci:
+  recipe_owner: HuiyingLi
```
Comment (akoumpa) [inline examples/llm_finetune/qwen/qwen2_5_7b_hellaswag_fp8.yaml]:
```suggestion
  recipe_owner: adil-a
```

File: examples/llm_finetune/qwen/qwen2_5_7b_instruct_chat.yaml [hunk 4]
```diff
@@ -114,4 +114,5 @@ lr_scheduler:
 #   name: <your_wandb_exp_name>
 #   save_dir: <your_wandb_save_dir>
 
-
+ci:
+  recipe_owner: HuiyingLi
```
Comment (akoumpa) [inline examples/llm_finetune/qwen/qwen2_5_7b_instruct_chat.yaml]:
```suggestion
  recipe_owner: adil-a
```

File: examples/vlm_finetune/gemma3/gemma3_vl_4b_cord_v2_megatron_fsdp.yaml [hunk 5]
```diff
@@ -96,3 +96,6 @@ freeze_config:
 #   wandb_entity:
 #   wandb_exp_name:
 #   wandb_save_dir:
+
+ci:
+  recipe_owner: BoxiangW
```
Comment (akoumpa) [inline examples/vlm_finetune/gemma3/gemma3_vl_4b_cord_v2_megatron_fsdp.yaml]:
```suggestion
  recipe_owner: ZhiyuLi-Nvidia
```

File: examples/llm_finetune/llama3_2/llama3_2_1b_hellaswag_hsdp.yaml [hunk 6]
```diff
@@ -94,4 +94,7 @@ optimizer:
 #   project: <your_wandb_project>
 #   entity: <your_wandb_entity>
 #   name: <your_wandb_exp_name>
-#   save_dir: <your_wandb_save_dir> 
+#   save_dir: <your_wandb_save_dir>
+
+ci:
+  recipe_owner: BoxiangW
```
Comment (akoumpa) [inline examples/llm_finetune/llama3_2/llama3_2_1b_hellaswag_hsdp.yaml]:
```suggestion
  recipe_owner: ZhiyuLi-Nvidia
```

File: examples/llm_finetune/llama3_2/llama3_2_1b_hellaswag_megatron_fsdp.yaml [hunk 7]
```diff
@@ -93,3 +93,6 @@ optimizer:
 #   entity: <your_wandb_entity>
 #   name: <your_wandb_exp_name>
 #   save_dir: <your_wandb_save_dir>
+
+ci:
+  recipe_owner: BoxiangW
```
Comment (akoumpa) [inline examples/llm_finetune/llama3_2/llama3_2_1b_hellaswag_megatron_fsdp.yaml]:
```suggestion
  recipe_owner: ZhiyuLi-Nvidia
```

File: examples/llm_finetune/llama3_2/llama3_2_1b_squad_megatron_fsdp.yaml [hunk 8]
```diff
@@ -87,3 +87,5 @@ optimizer:
 #   name: <your_wandb_exp_name>
 #   save_dir: <your_wandb_save_dir>
 
+ci:
+  recipe_owner: BoxiangW
```
Comment (akoumpa) [inline examples/llm_finetune/llama3_2/llama3_2_1b_squad_megatron_fsdp.yaml]:
```suggestion
  recipe_owner: ZhiyuLi-Nvidia
```

File: examples/llm_finetune/llama3_2/llama3_2_1b_squad_skypilot.yaml [hunk 9]
```diff
@@ -126,3 +126,6 @@ optimizer:
 lr_scheduler:
   lr_decay_style: cosine
   min_lr: 1.0e-6
+
+ci:
+  recipe_owner: Anakintano
```
Comment (akoumpa) [inline examples/llm_finetune/llama3_2/llama3_2_1b_squad_skypilot.yaml]:
```suggestion
  recipe_owner: adil-a
```

File: examples/llm_finetune/qwen/qwen3_moe_30b_lora.yaml [hunk 10]
```diff
@@ -100,4 +100,5 @@ optimizer:
   eps: 1e-8
 
 ci:
+  recipe_owner: yuhezhang-ai
```
Comment (akoumpa) [inline examples/llm_finetune/qwen/qwen3_moe_30b_lora.yaml]:
```suggestion
  recipe_owner: adil-a
```

File: examples/llm_finetune/qwen/qwq_32b_squad.yaml [hunk 11]
```diff
@@ -99,5 +99,6 @@ optimizer:
 #   save_dir: <your_wandb_save_dir> 
 
 ci:
+  recipe_owner: HuiyingLi
```
Comment (akoumpa) [inline examples/llm_finetune/qwen/qwq_32b_squad.yaml]:
```suggestion
  recipe_owner: adil-a
```

File: examples/llm_finetune/qwen/qwq_32b_squad_peft.yaml [hunk 12]
```diff
@@ -99,6 +99,7 @@ optimizer:
   # min_lr: 1.0e-5
 
 ci:
+  recipe_owner: HuiyingLi
```
Comment (akoumpa) [inline examples/llm_finetune/qwen/qwq_32b_squad_peft.yaml]:
```suggestion
  recipe_owner: adil-a
```

File: examples/llm_finetune/seed/seed_coder_8b_instruct_hellaswag_fp8.yaml [hunk 13]
```diff
@@ -116,3 +116,6 @@ optimizer:
 #   entity: <your_wandb_entity>
 #   name: <your_wandb_exp_name>
 #   save_dir: <your_wandb_save_dir>
+
+ci:
+  recipe_owner: HuiyingLi
```
Comment (akoumpa) [inline examples/llm_finetune/seed/seed_coder_8b_instruct_hellaswag_fp8.yaml]:
```suggestion
  recipe_owner: adil-a
```

**Review by akoumpa — APPROVED** [review]
Thanks a lot @thomasdhc
